### ISSUE TITLE
SystemProfile shows up for systems with an unsupported ABI

### ISSUE THREAD

**Issue by AWoloszyn** [google/gapid#3507]
If you plug an Android device in with an unsupported ABI, the SystemProfiling option shows up, but not the Graphics option, and there is no indication as to why.

**Comment by pau-baiget** [comment]
@lpy can you please verify if this is still happening?

**Comment by lpy** [comment]
Yes, this is still happening. If we do `bazel build --config=nodroid`, GAPID still gives `System Profile` as an option for an Android device.

There are two places we have the concept of unsupported ABI, one is inside https://github.com/google/gapid/blob/master/core/cc/supported_abis.h#L25, where we explicitly return what are the supported ABIs; the other is when we try to ensure we install gapid.apk https://github.com/google/gapid/blob/master/gapidapk/gapidapk.go#L53

Is there any place I miss? A quick thought is to plumb `supported_abis.h` to go land, otherwise we can disable `System Profile` if `EnsureInstalled` fails.
